playwright-e2e/dss/component/input.ts

```diff
@@ -79,4 +79,25 @@ export default class Input extends WidgetBase {
       await expect(this.page.locator('button:visible', { hasText: 'Saving' })).toBeHidden();
     }
   }
+
+  public async fillSimple(value: string): Promise<void> {
+    await this.toLocator().fill(value);
+    await this.toLocator().blur();
+  }
```

Comment:
Instead creating a new function, you could modify existing `fill()` to have a new optional param `blur?: boolean`